src/vs/workbench/services/extensions/electron-sandbox/electronExtensionService.ts

```diff
@@ -539,7 +541,7 @@ export abstract class ElectronExtensionService extends AbstractExtensionService
 			// fetch the remote environment
 			[remoteEnv, remoteExtensions] = await Promise.all([
 				this._remoteAgentService.getEnvironment(),
-				this._remoteAgentService.scanExtensions()
+				this._unifiedExtensionScannerService.scanExtensions(ScanLocation.Remote),
```

Comment:
Why is `_unifiedExtensionScannerService` is not used for scanning local extensions?